src/test/scala/com/cognite/sdk/scala/v1/TransformationsTest.scala

```diff
@@ -0,0 +1,214 @@
+// Copyright 2020 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+package com.cognite.sdk.scala.v1
+
+import cats.Id
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import com.cognite.sdk.scala.common._
+import com.cognite.sdk.scala.v1.FlatOidcCredentials._
+
+import java.time.temporal.ChronoUnit
+import java.time.Instant
+import scala.concurrent.duration.DurationInt
+import scala.util.control.NonFatal
+import java.util.UUID
+
+@SuppressWarnings(
```

Comment:
Shouldn't need any of those, I think. There's a reason we have the warnings, and it's not to just suppress them all, but sometimes they are wrong. ;)

Reply:
Only `NonUnitStatements` left, it's pretty "dumb" I think 😅  because it wants something like below instead
```scala
val _ = resCreates.size shouldBe transformationsToCreate.size
```
`AsInstanceOf` and `OptionPartial` can be removed if we sort out the thing with the GenericClient 😁

Reply:
I'm not sure it will if you do `val resCreates = client.transformations.createItems(Items(transformationsToCreate)).unsafeRunSync()` but let's see...